src/pages/plugins/index.js

```diff
@@ -104,28 +104,35 @@ function PluginComponent({ plugin }) {
   return (
     <div className="row padding-vert--md">
       <div className="col col--10 col--offset-1">
-        <div key={plugin.key} className={ plugin.deprecatedMessage ? `card ${classnames('card__deprecated', styles.pluginDeprecatedCard)}` : "card"}>
-          {!plugin.deprecatedMessage || (
+        {plugin.deprecatedMessage ? (
```

Comment:
Instead of creating an entirely new DOM structure, you should just create variables for the values that change when a package is deprecated.

Reply:
I created the variables name with deprecatedCard and will pull request, but need seems further review if it is correctly used.